scripts/install_dependencies.sh

```
@@ -1,12 +1,21 @@
 #!/bin/bash
 
 # Work at the root directory
-# Should have conanfile.py present there.
 cd "$(dirname "$0")"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)